assignment/src/main/java/nerdschool/bar/Pub.java

```diff
@@ -1,69 +1,7 @@
 package nerdschool.bar;
 
+// A copy for commenting code
+	
 public class Pub {
 
```

Comment:
When evaluating why the code is bad in details, we should assess the candidate ability to think about good principles and practices.
Examples :
- SOLID principle? Single Responsability... etc
- Code quality & craftmanship (DRY, YAGNI, KISS)
- Clear naming and readability (full, real, intention-revealing, meaningful names, with the proper case)
- Method and class sizes? The smaller and the more specialized the better
- Testing is crucial ! Anticipating/talking about tests and TDD is a good developer reflex